contents/docs/product-analytics/schema-management.mdx

```diff
@@ -0,0 +1,146 @@
+---
+title: Schema management
+sidebar: Docs
+showTitle: true
+availability:
+    free: full
+    selfServe: full
+    enterprise: full
+---
+
+Schema management lets you define and enforce the structure of your events using typed property groups. This helps document expected properties, provides type safety in your code, and makes your analytics implementation more maintainable.
+
+## Creating property groups
+
+Property groups are reusable collections of typed properties that can be attached to one or more events.
+
+1. Navigate to **Data Management** > **Property Groups**
+2. Click **New Property Group**
+3. Give your group a name (e.g., "Order Information")
+4. Add properties with their types:
+   - **Name**: The property key
+   - **Type**: String, Number, Boolean, or Object
+   - **Required**: Whether the property must be present
+   - **Description**: Help text for your team
+
+![Creating a property group](https://res.cloudinary.com/dmukukwp6/image/upload/w_1600,c_limit,q_auto,f_auto/create_property_group_476e273705.png)
+
+Property groups can be reused across multiple events, making it easy to maintain consistent schemas.
+
+## Creating events
```

Comment:
Should we create events first then properties? Then we can say, to edit existing events, skip ahead. Logically it's events -> Add Property Group -> create schema if need be.

Weak opinion, especially if you designed this flow other wise!